sphinx/source/docs/user_guide/contour.rst

```diff
@@ -0,0 +1,134 @@
+.. _userguide_contour:
```

Comment:
What do you you think about adding a more general section at the top level, e.g "Specialized Plots" or "Schematic Plots" and then adding  a sub-page for this underneath?  (similar to the "interactions" chapter structure) We could potentially also have a page for hexbin and related plots underneath, and leaves a place for additional specialized plots to go in the future. I am just worried that contour plots by itself is a bit narrow of a topic for an entire top-level User Guide chapter. cc @tcmetzger for thouthgs

Reply:
We could even potentially move map/geo plots underneath a general "specialized plots" chapter

Reply:
Also to clarify: I don't mean to move those other sections in this PR, I am just trying to motivate the rationale for the proposal.

Reply:
Yes, I think this is a good idea and I prefer "Specialized plots" to "Schematic plots".

I would classify the existing network graphs as well as map/geo data as similarly specialized and put them under the same heading. I have held off from doing this yet but am happy to do so whilst I am here.